core/tezos_operation.re

```diff
@@ -0,0 +1,48 @@
+open Helpers;
+open Crypto;
+
+[@deriving yojson]
+type internal_operation =
+  | Tezos_deposit({
+      destination: Tezos.Address.t,
```

Comment:
shouldn't this be just Address.t? we don't have anything else besides tz1 addresses on deku. But anyways to make it Address.t we would also need to modify tezos_interop.

Reply:
This is a Tezos_deposit, it comes from consensus.mligo and consensus.mligo accepts a KT1 address.

Reply:
Merged as this was a non blocking comment